pkg/tenant/projects.go

```diff
@@ -424,6 +433,15 @@ func (s *Server) ProjectUpdate(c *gin.Context) {
 	// Update all user provided fields
 	p.Name = project.Name
 
+	if project.Description != "" {
```

Comment:
I added this after realizing that user's may want to edit their project description.

Reply:
Agreed, good call!